src/jvmTest/kotlin/wizard/GeneratedComposeAppProjectTest.kt

```diff
@@ -163,6 +164,34 @@ class GeneratedComposeAppProjectTest {
             }
         )
         val dir = projectInfo.writeToDir(workingDir)
+        val devicesJson = checkCommand(
+            dir = dir,
+            command = listOf(
+                "xcrun",
+                "simctl",
+                "list",
+                "devices",
+                "available",
+                "-j"
+            )
+        )
+        val devicesList = Json.createReader(StringReader(devicesJson)).use { it.readObject() }
+            .getJsonObject("devices")
+            .let { devicesMap -> devicesMap.keys.map { devicesMap.getJsonArray(it) } }
+            .map { jsonArray -> jsonArray.map { it.asJsonObject() } }
+            .flatten()
+            .filter { it.getBoolean("isAvailable") }
+            .filter {
+                if (true.toString().equals(other = System.getenv("CI"), ignoreCase = true)) {
```

Comment:
is it the same as `System.getenv("CI").toBoolean()` ?

Reply:
It looks better